cle/backends/coff.py

```diff
@@ -0,0 +1,568 @@
+#!/usr/bin/env python3
```

Comment:
Probably shouldn't have a shebang on a file that isn't intended to be run as main

Reply:
Agreed, this file used to have a `main` but I forgot to pull the shebang